src/utility/cartFunction.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+const encodedToken =
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI4NDA1MjA4MS1hNjAwLTQ2YmQtYTNhZS0yZDljNTU2YTQ0NzgiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.gaqvCVkY0hv54te82TnyI8W3iLXNbRScUytBNaTE3GM";
```

Comment:
why you store the token this way, instead you can do auth functionality

Reply:
Auth functionality is yet to be done once it done encoded token will come from there